src/components/Presentation/DetailList.vue

```diff
@@ -0,0 +1,62 @@
+<template lang="pug">
+v-list.detail-list(v-if="rows.length", :style="{ backgroundColor: color }")
+  v-subheader {{ title }}
+  div(v-for="val, i in rows", :key="i")
+    v-list-item
+      slot(name="row", :datum="val")
+        v-list-item-content.py-1
+          span.secondary--text.text--darken-3.no-overflow {{ val }}
+    v-divider.mx-3(v-if="i < rows.length - 1")
+</template>
+
+<script>
+export default {
+  props: {
+    title: {
+      type: String,
+      required: true,
+    },
+    rows: {
+      type: Array,
+      required: true,
+    },
+    color: {
+      type: String,
+      default: 'inherit',
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+.detail-list {
+  &.v-list {
+    background: transparent;
+    max-height: 400px;
+    overflow: auto;
+  }
+
+  .v-list-item {
+    font-size: 13px;
+    height: inherit;
+    min-height: 28px;
+  }
+
+  .v-subheader {
+    font-size: 16px;
+    font-weight: 700;
+    height: 32px;
+  }
+
+  .no-overflow {
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+    width: 100%;
+  }
+
+  .v-btn__content {
+    justify-content: flex-start;
```

Comment:
Given that there are no `v-btn`s in this, this style rule doesn't belong here. I'm actually working on a proposed change that will remove all these rules, I'll push that up to this branch for you to review.

Reply:
Awesome, I hadn't really reviewed these styles since I wrote them back in February.  Looks like most of them are reproducible with Vuetify conventions.  I'll wait to make any changes until I've seen yours.

Reply:
4fe54a9 is my suggested change, it has the benefit of removing all the CSS from the reusable components.

I think there's an argument to be made that the entire `v-list-item` inside DataDetails should be inside of the slot rather than just its contents, to allow for things like the `to` attribute, or any other attributes of the `v-list-item` that someone might want to control from outside. What do you think?

Reply:
To be honest, I'm not sure `v-list` is appropriate at all here.   We aren't really using any of its features except for the buttony thing. `v-list` produces lists that generally feel more "important" than simple lists of text (navigation bars and the like).

I think we should keep your changes for now though, until some more advanced UI need presents itself.